contentlayer.config.ts

```diff
@@ -55,4 +111,19 @@ export const Transcript = defineDocumentType(() => ({
 export default makeSource({
   contentDirPath: path.join(process.cwd(), "public", "bitcoin-transcript"),
   documentTypes: [Transcript],
+  contentDirExclude: [
+    ".github",
+    ".gitignore",
+    "LICENSE.md",
+    "README.md",
+    "STYLE.md",
+    "twitter_handles.json",
+    ".json",
```

Comment:
Nice, I forgot the `contentDirExclude` when I did the refactoring.

Does `".json"` means that it's excluding all the JSON files?

Reply:
No it doesn't, this and other things in the `contentDirExclude` array just excludes files at the root directory

Reply:
let me tag this comment to the pr in prod